logdna/client_test.go

```diff
@@ -0,0 +1,91 @@
+package logdna
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestClient_View(t *testing.T) {
+	body := make(map[string](interface{}))
```

Comment:
this appears to be unused (other than decoding the request body into it, nothing is done with it after that)

Reply:
@mdeltito I'm moving adding httptest to another Jira task/PR (see https://logdna.atlassian.net/browse/LOG-7937). This one already includes preset alerts and documentation updates so don't want to have too many things going on simultaneously in this one.

Reply:
agreed, that sounds like a plan